apps/storefront/src/shared/service/request/fetch.ts

```diff
@@ -1,51 +1,25 @@
-import { snackbar } from '@/utils';
-
-import { RequestType } from './base';
-
 const originFetch = window.fetch;
 
-const responseResult = (path: string, res: any, resolve: any, init: any) => {
-  if (path.includes('current.jwt')) return res.text();
-
+const responseResult = (res: any, resolve: any, init: any) => {
```

Comment:
It seems that you remove it `path` param instad of `init`, that's causing a problem

Reply:
🤦 we need tests